test/adjoint-rotate.jl

```diff
@@ -25,32 +24,27 @@ end
         image3 = rand(T, nx, nx, 4)
         result = similar(image3)
         imrotate!(result, image3, π/6, plan)
-        @test maximum(result) ≤ 1.1
-        imrotate_adj!(result, image3, -π/6, plan)
-        @test maximum(result) ≤ 1.1
+        @test maximum(result) ≤ 1.01
+        imrotate_adj!(result, image3, π/6, plan)
+        @test maximum(result) ≤ 1.5
```

Comment:
that's something I have not figured out yet, why the 2d interpolation adjoint enlarge the range of pixels so much?